src/assets/wise5/components/graph/graph-student/graph-student.component.ts

```diff
@@ -2033,6 +2032,30 @@ export class GraphStudent extends ComponentStudent {
     this.previousTrialIdsToShow = copy(this.trialIdsToShow);
   }
 
+  /**
+   * Check if two arrays contain the same values. This is commonly used to
+   * check if two arrays of ids contain the same values. The order of the
+   * elements is not compared, only the actual values. This means the elements
+   * can be in different orders but still contain the same values.
+   * Example:
+   * array1=['1234567890', 'abcdefghij']
+   * array2=['abcdefghij', '1234567890']
+   * If these two arrays are passed in as the two arguments, this function
+   * will return true.
+   * Note: This may only work if the elements are strings, numbers or
+   * booleans. If the elements are objects, this function may or may not work.
+   * @param array1 an array of strings, numbers, or booleans
+   * @param array2 an array of strings, numbers, or booleans
+   * @return whether the arrays contain the same values
+   */
+  private arraysContainSameValues(array1: any[], array2: any[]): boolean {
```

Comment:
Looks like array1 and array2 will always be string arrays? If so, change ```any[]``` to ```string[]``` and update the function comments.

```suggestion
  /**
   * Check if two string arrays contain the same values. The order of the
   * elements is not compared, only the actual values. This means the elements
   * can be in different orders but still contain the same values.
   * Example:
   * array1=['1234567890', 'abcdefghij']
   * array2=['abcdefghij', '1234567890']
   * If these two arrays are passed in as the two arguments, this function
   * will return true.
   * @param array1 an array of strings
   * @param array2 an array of strings
   * @return whether the arrays contain the same values
   */
  private arraysContainSameValues(array1: string[], array2: string[]): boolean {
```